Packs/Core/Integrations/CortexCoreIR/CortexCoreIR_test.py

```diff
@@ -366,3 +367,178 @@ def test_get_distribution_url_command_without_download_not_supported_type():
+        "is_raw_command": True
+    }
+    reformate_args(args)
+    params = json.loads(args["parameters"])
+    assert params["commands_list"] == ["dir, hostname"]
+    assert args["is_core"] is True
+    assert args["script_uid"] == "a6f7683c8e217d85bd3c398f0d3fb6bf"
+
+
+@pytest.mark.parametrize("separator", [',', '/', '|'])
+def test_reformat_args_separators(separator):
+    """
+    Given:
+        - is_raw_command argument set to False (default) and a chosen separator.
+    When:
+        - Calling `reformate_args` with each of the separators options.
+    Then:
+        - Verify that commands_list split by the chosen separator.
+    """
+    args = {
+        "command": f"dir{separator}hostname",
+        "is_raw_command": False,
+        "command_separator": separator
+    }
+    reformate_args(args)
+    params = json.loads(args["parameters"])
+    assert params["commands_list"] == ["dir", "hostname"]
+
+
+def test_reformat_args_powershell_command_formatting():
+    """
+    Given:
+        - command_type argument set to powershell.
+    When:
+        - Calling `reformate_args` with command_type=powershell.
+    Then:
+        - Verify command at commands_list reformated from 'command' to 'powershell -Command "command"'.
+    """
+    args = {
+        "command": "Get-Process",
+        "command_type": "powershell",
+        "is_raw_command": True
+    }
+    reformate_args(args)
+    params = json.loads(args["parameters"])
+    assert params["commands_list"] == ['powershell -Command "Get-Process"']
+
+
+def test_reformat_output():
+    """
+    Given:
+        - Response from polling command.
+    When:
+        - Calling `reformat_output` with response.
+    Then:
+        - Verify output unify all duplicated data.
+        Instead of a list with element for each commmand, we'll have an element for each endpoint.
+    """
+    from CortexCoreIR import reformat_output
+    mock_res = CommandResults(outputs_prefix='val', outputs=load_test_data("./test_data/execute_command_response.json"))
+    reformat_output([mock_res])
+    excepted_output = {
+        "action_id": 1,
+        "results": [
+            {
+                "endpoint_name": "name",
+                "endpoint_ip_address": ["218.92.0.29"],
+                "endpoint_status": "STATUS_010_CONNECTED",
+                "domain": "domain.name",
+                "endpoint_id": "a6f7683c8e217d85bd3c398f0d3fb6bf",
+                "execution_status": "COMPLETED_SUCCESSFULLY",
+                "executed_command":
+                [
+                    {
+                        "command": "echo",
+                        "failed_files": 0,
+                        "retention_date": None,
+                        "retrieved_files": 0,
+                        "standard_output": "output",
+                        "command_output": []
+                    },
+                    {
+                        "command": "echo hello",
+                        "failed_files": 0,
```

Comment:
let's use dummy values instead of IPS and endpoint IDs (relevant to this test and the next one).